### PR TITLE
ci: Create releases with GitHub actions

### LINKED ISSUE #27: Setup automatic build system
Standalone executables for the three major OS.

### PR DESCRIPTION
**Pull request by g3n35i5**
This resolves #27

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ jobs:
             PYINSTALLER_ARGS: "--onefile"
           - os: macos-latest
             APP_NAME: "USDB Syncer.app"
```
Comment (bohning) [inline .github/workflows/release.yaml:19]: APP_NAME should be "USDB Syncer", right? The `.app` is appended as part of the `pyinstaller --windowed` process, I believe.

File: .vscode/settings.json [hunk 2]
```diff
@@ -23,7 +24,5 @@
   },
   "python.languageServer": "Default",
   "python.analysis.typeCheckingMode": "basic",
-  "editor.rulers": [
-    88
-  ]
-}
\ No newline at end of file
+  "editor.rulers": [120]
```
Comment (RumovZ) [inline .vscode/settings.json:27]: Was there any particular reason you changed this? It does'nt conform to our configured formatting rules.

**Review by RumovZ — CHANGES_REQUESTED** [review]
Great job, works like a charm on Windows!

Do you know why Windows Defender would suddenly warn about running the app? It didn't with the previous exe.
It would also be convenient if we could skip the archiving on Windows and Linux. But not a big deal.

**Review by RumovZ — APPROVED** [review]

**Comment by g3n35i5** [comment]
> Great job, works like a charm on Windows!
> 
> Do you know why Windows Defender would suddenly warn about running the app? It didn't with the previous exe. It would also be convenient if we could skip the archiving on Windows and Linux. But not a big deal.

I would leave this as a ZIP for two reasons: It saves storage space and unifies all platforms. Otherwise you always have to consider special conditions "If on Windows/Linux then don't zip, otherwise do".

On the subject of Windows Defender: I think that the warning should always come, because the exe is not signed?

**Pull request merged**